snap/snapcraft.yaml

```diff
@@ -237,6 +239,8 @@ apps:
     adapter: none
     after:
       - security-bootstrap-redis
+      # have consul role generated first for this service
```

Comment:
How about  `This generates the consul role for this service before the service starts'. Here and other same comments

Reply:
fixed